<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cost_serch</class>
 <widget class="QDialog" name="cost_serch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>375</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>1</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>지출 비용 검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cost_reg_return_Button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_Button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>초기화</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>61</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>검색어 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serch_word_widget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>62</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="cost_serch_tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>651</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>일자</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>항</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>목</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>세목</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>적요</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="cost_serch_Button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>41</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>겸색년도 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serch_year_widget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>41</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>년도</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serch_year_widget</tabstop>
  <tabstop>serch_word_widget</tabstop>
  <tabstop>cost_serch_tableWidget</tabstop>
  <tabstop>cost_serch_Button</tabstop>
  <tabstop>reset_Button</tabstop>
  <tabstop>cost_reg_return_Button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cost_serch_Button</sender>
   <signal>clicked()</signal>
   <receiver>cost_serch</receiver>
   <slot>cost_reg_serch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset_Button</sender>
   <signal>clicked()</signal>
   <receiver>cost_serch</receiver>
   <slot>serch_reset_Button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cost_reg_return_Button</sender>
   <signal>clicked()</signal>
   <receiver>cost_serch</receiver>
   <slot>cost_reg_return()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cost_reg_return()</slot>
  <slot>serch_reset_Button()</slot>
  <slot>cost_reg_serch()</slot>
 </slots>
</ui>
